src/Umbraco.Core/Persistence/Constants-Locks.cs

```diff
@@ -85,5 +85,10 @@ public static class Locks
         ///     Long-running operations.
         /// </summary>
         public const int LongRunningOperations = -344;
+
+        /// <summary>
+        ///     The entire element tree, i.e. all element items.
+        /// </summary>
+        public const int ElementTree = -345;
```

Comment:
```suggestion
        public const int ElementTree = -347;
```

A couple more of these have since been added (document URLs and cache versions).